Source/Diagnostics/Diagnostics.cpp

```diff
@@ -57,7 +57,30 @@ Diagnostics::BaseReadParameters ()
     // Query list of grid fields to write to output
     bool varnames_specified = pp_diag_name.queryarr("fields_to_plot", m_varnames);
     if (!varnames_specified){
-        m_varnames = {"Ex", "Ey", "Ez", "Bx", "By", "Bz", "jx", "jy", "jz"};
+#if defined(WARPX_DIM_RZ)
+    bool is_rz = true;
+#else
+    bool is_rz = false;
```

Comment:
Is there a run-time equivalent of WARPX_DIM_RZ? This may be clunky

Reply:
Yes, it's `geometry.dims`
Ref.: https://warpx.readthedocs.io/en/latest/usage/parameters.html#setting-up-the-field-mesh